### ISSUE TITLE
convert from PropertySet to Property

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/fraction-comparison#36]
PropertySet is deprecated and should be replaced by Property, see phetsims/axon#102. Since this sim hasn't been published yet, this should be done before RC testing and publishing.

**Comment by veillette** [comment]
This was covered in 6ad8db2e206b258a2dfb68e0a7f7b8e6c868a1d4,
and 2311ece945528c0cc2cab817f013b64ceb7e9c25.

Assigning to @samreid for review.

**Comment by veillette** [comment]
The simulation works well with  ea and fuzzMouse enabled. The reset button works as well.

**Comment by samreid** [comment]
The changes look great!  I have a few very minor recommendations as you proceed:

(1) some annotations look like this:

```js
    // @public {Property.<string>} one of start/dragging/compare
    this.stateProperty = new StringProperty( 'start' );
```

It should also be noted when the property is read-only, like so:

```js
// @public (read-only) {Property.<string>} one of start/dragging/compare
```

In this case, it means that stateProperty is never reassigned.

(2)
In this code:
```js
  // constant
  var VALID_REPRESENTATION_VALUES = [
    'horizontal-bar',
    'vertical-bar',
    'circle',
    'chocolate',
    'different-sized-circles' ];
```

// constant should be labeled //constants, even if there is only one of them for now.

**Comment by samreid** [comment]
On second thought, I'll close this issue because the PropertySet fix looks great, and I'll move the preceding comment to a new issue.
